### PR TITLE
Add popin when saving content

### PR DESCRIPTION
**Pull request by fkroockmann**
https://github.com/backbee/backbee-standard/issues/90

### CODE REVIEW

File: src/tb/apps/content/controllers/main.controller.js [hunk 1]
```diff
@@ -127,8 +128,25 @@ define(
...
+                    translator = req('component!translator'),
+                    nbContents = SaveManager.getContentsToSave().length;
+
+                popin.setTitle(translator.translate('save_draft'));
+                popin.addButton('Ok', popin.hide);
+
+                if (nbContents > 0) {
+                    popin.mask();
+                    SaveManager.save().done(function () {
+                        popin.setContent(nbContents + ' ' + translator.translate((nbContents === 1) ? 'content_saved_sentence' : 'content_saved_sentence_plural'));
```
Comment (mickaelandrieu) [inline src/tb/apps/content/controllers/main.controller.js]: ``translator.translate((nbContents === 1)`` =>  translate() is support to receive a string

File: src/tb/i18n/fr_FR/global.json [hunk 2]
```diff
@@ -19,6 +19,8 @@
     "change_password": "Changer de mot de passe ?",
     "clear_all": "Tout supprimer",
     "content_management": "Gestion de contenus",
+    "content_saved_sentence": "contenu modifié and et sauvegardé en tant que brouillon",
```
Comment (mickaelandrieu) [inline src/tb/i18n/fr_FR/global.json]: ty for french translations :+1:

File: src/tb/i18n/fr_FR/global.json [hunk 3]
```diff
@@ -65,6 +67,7 @@
     "name": "Nom",
     "new_page": "Nouvelle page",
     "next": "Suivant",
+    "no_content_save": "Aucun contenu à sauvegardé",
```
Comment (mickaelandrieu) [inline src/tb/i18n/fr_FR/global.json]: à sauvegarder

File: src/tb/i18n/fr_FR/global.json [hunk 4]
```diff
@@ -19,6 +19,8 @@
     "change_password": "Changer de mot de passe ?",
     "clear_all": "Tout supprimer",
     "content_management": "Gestion de contenus",
+    "content_saved_sentence": "contenu modifié and et sauvegardé en tant que brouillon",
+    "content_saved_sentence_plural": "contenus modifiés and et sauvegardés en tant que brouillon",
```
Comment (mickaelandrieu) [inline src/tb/i18n/fr_FR/global.json]: and ? :)

File: src/tb/apps/content/controllers/main.controller.js (hunk 1 above)
Comment (fkroockmann) [inline src/tb/apps/content/controllers/main.controller.js]: Can you explain ?

**Comment by fkroockmann** [comment]
thanks for review @mickaelandrieu

**Pull request merged**